### DISCUSSION TITLE
Stop keyboard from going full screen in portrait mode on big screen

### DISCUSSION THREAD

**Discussion by willbou1** [element-hq/element-meta#1693] [Ideas]
I just got myself a Samsung Z Fold 4 which made me stumble on a little problem when using Element in portrait mode on the main screen. When I start editing my message, the keyboard opens in fullscreen which is a big waste of space. By that, I mean that all the remaining space on top of the keyboard becomes a big text box for the currently edited message.

Thank you to all the Element team! You're doing an amazing job!

**Comment by t3chguy** [comment]
This is an issue for https://github.com/vector-im/element-android - I don't have permission to transfer there
